QueueProcessors/restart.sh

```diff
@@ -6,18 +6,33 @@
 #### WARNING: If there's currently reduction runs "processing" this may 
 #### cause them to get stuck in the "processing" state.
 
+ROOT_DIR="$(dirname "$0")"
+
+## Autoreduction Processor
 pkill -9 -f "python AutoreductionProcessor/autoreduction_processor_daemon.py start" &&
```

Comment:
This commit breaks this because this process is not found now. Need to add wild card `.*` infront of `autoreduction_processor_daemon.py` so it doesn't matter where the script is ran from.

Reply:
Ah yes I missed these thanks.